ruby/lib/helper_functions.rb

```diff
@@ -0,0 +1,21 @@
+
+  
+def quality_limit?(item)
+  item.quality < 50
+end
```

Comment:
I really like pulling out helper functions.  I like the attempt.  Few things I want to talk about live -- naming, and the decisions we all wrestle with when creating helpers.

`reached_quality_limit?`